examples/simple_axi_interconnect/simple_axi_interconnect.h

```diff
@@ -0,0 +1,294 @@
+/*
+ * ace_axi_adapt.h
+ *
+ *  Created on: Aug 13, 2022
+ *      Author: tientr
+ */
+#ifndef __SIMPLE_AXI_INTERCONNECTION__
+#define __SIMPLE_AXI_INTERCONNECTION__
+
+#include "generic_extension.h"
+#include <axi/axi_initiator.h>
+#include <axi/axi_target.h>
+#include <axi/axi_tlm.h>
+#include <fstream>
+#include <iostream>
+#include <map>
+
+template <uint64_t SOCKET_WIDTH, uint64_t MASTER_NUM, uint64_t SLAVE_NUM>
```

Comment:
Sorry, I don't know the context of this pull request. 
Why is an example module so flexibly configurable (with templates and file input)? I would expect that example modules should be as simple as possible. Or should this become an SCC component in the future?

Reply:
sorry i lost my account. So, late reponse,
that is example to build a simple system bus, and support configure address map without compile.
i think that will be useful in a big system. 
anyway , thanks for your feedback.